StRoot/StFttClusterMaker/StFttClusterMaker.cxx

```diff
@@ -0,0 +1,429 @@
+/***************************************************************************
```

Comment:
This is a new maker.  Review for this will be a little more detailed.

Reply:
Right, two reviewers have been identified (one agreed, one waiting). But your comments are appreciated nevertheless.